test/powershell/Modules/Microsoft.PowerShell.Utility/Pester.Commands.Cmdlets.Json.Tests.ps1

```diff
@@ -1455,4 +1454,43 @@ Describe "Json Bug fixes"  -Tags "Feature" {
     {
         RunJsonTest $testCase
     }
+
+    It "ConvertFrom-Json can parse an array of PSObjects as a single string." {
+
```

Comment:
Can the variations be converted to use -TestCase for It? So if one of the Should fails the others are still run.